### PR TITLE
Release view-app-changes toggle

### PR DESCRIPTION
**Pull request by sravfeyn**
## Product Description

https://dimagi-dev.atlassian.net/browse/SC-2476 

This moves the VIEW_APP_CHANGES toggle to a Advanced paid privilege.

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_view_app_diff_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        privileges.VIEW_APP_DIFF,
+        skip_edition='Paused,Community,Standard',
```
Comment (SmittieC) [inline corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py:16]: I don't have much experience with the plans yet, but should we not include the `pro`plan here as well?

**Comment by SmittieC** [comment]
I think the description should mention `to an advanced paid privilege` rather than `to a standard paid privilege` :smile:

File: corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py (hunk 1 above)
Comment (kaapstorm) [reply corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py:16]: If this is for Standard Plan, shouldn't `skip_edition` be `'Paused,Community'`?

**Review by kaapstorm — COMMENTED** [review]
Code is good. Just check what plan this is for.

* PR description says Standard Plan
* Jira ticket says Advanced Plan
* DB migration implies Pro Plan

File: corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/accounting/migrations/0066_add_view_app_diff_priv.py]: @kaapstorm @SmittieC 

Sorry my bad on Plan confusion, this is supposed to be under Advanced Plan, so I have updated it as such

**Comment by sravfeyn** [comment]
@kaapstorm @SmittieC Bumping to see if this is g2g

**Review by SmittieC — APPROVED** [review]
LGTM :rocket:

**Pull request merged**